spec/rails_helper.rb

```diff
@@ -2,15 +2,15 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 # Prevent database truncation if the environment is production
-abort("The Rails environment is running in production mode!") if Rails.env.production?
+abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'spec_helper'
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
-require "simplecov"
+require 'simplecov'
 SimpleCov.start
-if ENV["CI"] == "true"
-  require "codecov"
+if ENV['CI'] == 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.